src/yalexs_ble/lock.py

```diff
@@ -206,7 +206,7 @@ async def _handle_missing_characteristic(self, char_uuid: str) -> None:
 
     def _internal_state_callback(self, state: bytes) -> None:
         """Handle state change."""
-        _LOGGER.debug("%s: State changed: %s", self.name, state.hex())
+        _LOGGER.info("%s: State changed: %s", self.name, state.hex())
```

Comment:
```suggestion
        _LOGGER.debug("%s: State changed: %s", self.name, state.hex())
```

This is expected to happen under normal operation. Let's leave it at debug